wiki/Silence/en.md

```diff
@@ -10,17 +6,35 @@ stub: true
 
 Users who are silenced by moderators cannot do the following:
 
-- Edit their userpage
-- Change their avatar/profile details <!-- unconfirmed on new site, but was a thing on old -->
-- Submit [beatmaps](/wiki/Beatmap)
-- Send/receive private messages
+- Use [chat](/wiki/Chat_Console), both in-game and on the web site
+- Send and receive private messages
 - Post on any of the osu! forums
+- Comment on [beatmaps](/wiki/Beatmap)
+- Take part in [beatmap discussions](/wiki/Beatmap_Discussion)
+- Edit userpage
+- Change avatar and profile details
+- Submit and update beatmaps
+- Partake in [multiplayer](/wiki/Multi) games
+
+In addition to that, all messages of a silenced user are removed from all chat channels and private messages [1](https://blog.ppy.sh/post/38114063519/this-week-in-osu-5 "ppy blog").
+
+{{{ Infobox
+![](img/silence-profile.png "An example of silence record in profile page")
+
+![](img/silence-ingame.png "An example of silence warning shown in the chat")
+}}}
+
+After a user is silenced, they will see warning in place of the chat input field, saying when will the silence wear off. The silence record can also be observed at the bottom of the profile on the web site, from which it will disappear after a couple of weeks.
```

Comment:
```suggestion
After a user is silenced, an alert will be displayed in place of the chat input field, saying when the silence will wear off. The silence record can also be observed at the bottom of the profile on the website, from which it will disappear after 28 days.
```